### PR TITLE
Add slugs to configs

### PR DESCRIPTION
**Pull request by SaschaMann**
Bash worked 🤷 

```bash
for d in */; do d=${d/\//}; head -n 2 "$d/config.json" > tmp.json; echo "  \"slug\": \"$d\"," >> tmp.json; tail -n +3  "$d/config.json" >> tmp.json; mv tmp.json "$d/config.json"; done
```

### CODE REVIEW

File: languages/ruby/config.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "language": "Ruby",
   "slug": "ruby",
+  "slug": "ruby",
```
Comment (SaschaMann) [inline languages/ruby/config.json:4]:
```suggestion
```

File: languages/j/config.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "track_id": "j",
```
Comment (SaschaMann) [inline languages/j/config.json:2]: Whats this?

Comment (iHiD) [reply languages/j/config.json:2]: Some strange artifact.

**Comment by iHiD** [comment]
@SaschaMann Thanks so much! :)

**Comment by ErikSchierboom** [comment]
Great work @SaschaMann!

**Pull request merged**